server/routes/events.js

```diff
@@ -25,74 +25,146 @@ const isValidAccessContext = ({ accessContexts, userSchool, school }) => {
     }
 }
 
+/*
+    Helper function to ensure user has access to a given school
```

Comment:
Does this comment match what this function does?

Reply:
Hmmmm, you have a point sir.